indico/htdocs/js/custom/dropzone.js

```diff
@@ -63,18 +63,22 @@
                     }
                 }
 
-                $button.on('click', function(e) {
-                    e.preventDefault();
-                    e.stopPropagation();
-                    $button.prop('disabled', true);
-                    $.each(self.getRejectedFiles(), function(index, file) {
-                        self.removeFile(file);
-                    });
-                    if (self.getQueuedFiles().length) {
-                        $dz.find('.dz-progress').show();
-                        self.processQueue();
-                    } else if (options.submitIfEmpty) {
-                        $form.submit();
+                $form.on('submit', function(e) {
```

Comment:
Make sure to test all the places where we use a dropzone to ensure it still works fine